src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountSmall = 20;
```

Comment:
Consider using more descriptive constant names for 'discountSmall' and 'discountBig', such as 'discountFor3Days' and 'discountFor7Days'.